src/Make/AbstractMakeTimestamp.php

```diff
@@ -87,7 +87,8 @@ private function generatePdf(): string
             true, // PDF/A always for timestamp pdf
         );
         $log = (new Logger('elabftw'))->pushHandler(new ErrorLogHandler());
-        $MakePdf = new MakeTimestampPdf($log, $MpdfProvider, $this->Entity, array($this->Entity->id));
+        $includeChangelog = true;
```

Comment:
I'm guessing here you assigned true to the var so it would be easier to understand what is the meaning of the `true` in constructor below, right? While I can understand this approach, I prefer to avoid useless lines of code and as such ask you to simply add `true` in the constructor.